File: C/PRAK404-2210817310005-IkhsanGymnastiar.c
#include <stdio.h>
int main(){
          float opsi, b1, b2;
          while (1){
          printf("pilih program\n");printf("1. penjumlahan\n");printf("2. pengurangan\n");printf("3. perkalian\n");printf("4. penjumlahan\n");printf("5. exit\n");printf("Masukkan pilihan : "); scanf("%f", &opsi);
          if(opsi ==5){
                    printf("Terimakasih, telah menggunakan kalkulator Ikhsan Gymnastiar\n");break;
          }else if(opsi>5 ){
                    printf("Input anda salah, silahkan coba lagi\n");continue;}
          printf("Masukkan Bilangan Pertama : "); scanf("%f",&b1);
          printf("Masukkan Bilangan Kedua : "); scanf("%f",&b2);
          if(opsi == 1){
                    printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", b1, b2, b1+b2);
          }else if(opsi == 2){
                     printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", b1, b2, b1-b2);
          }else if(opsi == 3){
                     printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", b1, b2, b1*b2);
          }else if(opsi == 4){
                     printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", b1, b2, b1/b2);}}
return 0;}

File: Python/PRAK404-2210817310005-IkhsanGymnastiar.py
while True :
      print("pilih program\n 1. penjumlahan\n 2. pengurangan\n 3. perkalian\n 4. penjumlahan\n 5. exit\n") 
      opsi =int(input("Masukkan Pilihan : "))
      if opsi > 5 :
          print("Input anda salah, silahkan coba lagi")
          continue
      elif opsi == 5 :
            print("Terimakasih, telah menggunakan kalkulator Ikhsan Gymnastiar")
            break
      else :
          b1 =float(input("masukkan nilai Pertama : "))
          b2 =float(input("masukkan nilai Kedua : "))   
          if opsi == 1 :
                    print("Hasil penjumlahan antara {:.2f} dengan {:.2f} adalah {:.2f}".format(b1, b2, b1+b2))
          elif opsi == 2 :
                     print("Hasil pengurangan antara  {:.2f} dengan {:.2f} adalah {:.2f}".format(b1, b2, b1-b2))
          elif opsi == 3 :
                     print("Hasil perkalian antara  {:.2f} dengan {:.2f} adalah {:.2f}".format(b1, b2, b1*b2))
          elif opsi == 4 :
                     print("Hasil pembagian antara  {:.2f} dengan {:.2f} adalah {:.2f}".format(b1, b2, b1/b2))